src/components/MarkdownEditor/EditorToolbar.tsx

```diff
@@ -0,0 +1,312 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  $getSelection,
+  $isRangeSelection,
+  $isRootOrShadowRoot,
+  ElementNode,
+  RangeSelection,
+  TextNode,
+  FORMAT_TEXT_COMMAND,
+  SELECTION_CHANGE_COMMAND,
+  COMMAND_PRIORITY_CRITICAL,
+  COMMAND_PRIORITY_NORMAL,
+  KEY_MODIFIER_COMMAND,
+} from 'lexical';
+import styled from '@emotion/styled';
+import { $isAtNodeEnd } from '@lexical/selection';
+import { $isLinkNode, TOGGLE_LINK_COMMAND } from '@lexical/link';
+import {
+  $isListNode,
+  INSERT_ORDERED_LIST_COMMAND,
+  INSERT_UNORDERED_LIST_COMMAND,
+  ListNode,
+  REMOVE_LIST_COMMAND,
+} from '@lexical/list';
+import {
+  $findMatchingParent,
+  mergeRegister,
+  $getNearestNodeOfType,
+} from '@lexical/utils';
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { IconButtonV2 } from '@ndla/button';
+import { colors, misc, spacing } from '@ndla/core';
+import {
+  Bold,
+  Italic,
+  Link,
+  ListCircle,
+  ListNumbered,
+} from '@ndla/icons/editor';
+import { ADD_LINK_COMMAND } from './FloatingLinkEditorPlugin';
+
+const osCtrl = (key: string) => {
```

Comment:
Litt skeptisk til denne. Tanken er at vi skal vise `⌘+B` på Mac, `Ctrl+B` på windows/linux og ingenting på mobil. Mistenker at det kommer til å være vanskelig å være inklusiv nok her. Vet f.eks ikke hvordan Chromebook kommer til å håndtere dette.

Reply:
Kanskje det bare kan fjernes helt? 🤔  Ettersom det er helt standard shortcuts for de operasjonene...

Fører også til en advarsel i konsoll som isåfall burde fikses: _Prop `aria-label` did not match. Server: "Legg til fet skrift " Client: "Legg til fet skrift ⌘+b"_

Reply:
Personlig synes jeg det er bedre at dem er der på noen og ikke alle, enn at de ikke er på noen i det hele tatt. Selv om de er ganske standard, KAN det tenkes at vi vil kunne ha noen knapper som støtter snarveier og noen som ikke gjør, og da vil visning av snarveier der de støttes hjelpe med å formidle det til brukerne av editoren. Undersøkte Chromebook litt, og har lagt ved forslag for å inkludere det også.